### PR TITLE
Fixes example snippets in Quadrature.swift

### PR DESCRIPTION
**Pull request by valeriyvan**
Fixes example snippets in Quadrature.swift

### CODE REVIEW

File: stdlib/public/Darwin/Accelerate/Quadrature.swift [hunk 1]
```diff
@@ -46,6 +46,7 @@
 ///         }
 ///     }
 ///
+///     let diameter = 25.0
 ///     let vRresult = quadrature.integrate(over: 0.0 ... diameter,
```
Comment (CodaFi) [inline stdlib/public/Darwin/Accelerate/Quadrature.swift:50]: I think this was meant to match the example above, so just stating `0.0 ... 25.0` is fine.

Comment (valeriyvan) [reply stdlib/public/Darwin/Accelerate/Quadrature.swift]: Fixed

**Comment by CodaFi** [comment]
@swift-ci smoke test

**Review by CodaFi — APPROVED** [review]

**Comment by CodaFi** [comment]
⛵

**Pull request merged**